src/tests/Calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly add tests that simulate user interaction events like, for example in this component, performing a calculation. You can do this using the react testing library you installed. You visit this link for an idea of how to go about that: [https://www.robinwieruch.de/react-testing-library/](https://www.robinwieruch.de/react-testing-library/)